contracts/deploy/00-home-chain-arbitration.ts

```diff
@@ -39,6 +39,8 @@ const deployArbitration: DeployFunction = async (hre: HardhatRuntimeEnvironment)
     log: true,
   });
 
+  await deploy("ArbSysMock", { from: deployer, log: true });
```

Comment:
We don't want to deploy this on a live network, only on the Hardhat network. Also `ArbSysMock` is related to bridging so it move to the `02-home-chain.ts` script.